hbase-http/pom.xml

```diff
@@ -169,6 +169,48 @@
       <artifactId>log4j-slf4j-impl</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.directory.server</groupId>
```

Comment:
Just a comment.
I am a little concerned to include apache directory server. It's a big project with lots of dependencies, and there's been no release over 3 years. Very likely this dependency is going to to introduce transitive dependency versions know to be vulnerable. The apache directory server quarterly report last October mentioned a new release is coming up but nothing happens yet.

I understand this is for test scope only, so this is not a real issue.